json-schema-provider/pom.xml

```diff
@@ -46,6 +46,10 @@
             <groupId>com.fasterxml.jackson.datatype</groupId>
             <artifactId>jackson-datatype-jsr310</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.datatype</groupId>
+            <artifactId>jackson-datatype-joda</artifactId>
+        </dependency>
```

Comment:
Is this needed for swagger?

Reply:
No but it seems to be missing in the master branch right now.